File: java/src/TestExample.java
import static org.junit.Assert.*;
import org.junit.Test;

public class TestExample {

    @Test(timeout= 1000)
    public void test1() {
        Example e = new Example();
        assertEquals(55, e.sum(1, 10));
        assertEquals(1, e.sum(1, 1));
    }
}

File: scala/src/TestExample.scala
import org.testng.AssertJUnit._
import org.testng.annotations._

class TestExample {

    @Test(timeOut= 1000L)
    def test1(): Unit = {
        val e = new Example()
        assertEquals(55, e.sum(1, 10))
        assertEquals(1, e.sum(1, 1))
    }
}